seatunnel-core/seatunnel-core-flink/src/main/java/org/apache/seatunnel/core/flink/utils/CommandLineUtils.java

```diff
@@ -35,4 +42,44 @@ public static FlinkCommandArgs parseFlinkArgs(String[] args) {
             .parse(args);
         return flinkCommandArgs;
     }
+
+    public static FlinkCommandArgs parseCommandArgs(String[] args, FlinkJobStarter starter) {
```

Comment:
You should directly enhance the method in line 37, now you can remove the method in line 37.